sh: bower: command not found

Hello ...  I was updating  dynamodb-admin and the post install fails because I dont have bower... 
So if its on the post-install does that make bower a dependency not a devDependency ?

```
> dynamodb-admin@1.13.0 postinstall /Users/msaglietto/.nvm/versions/node/v6.9.2/lib/node_modules/dynamodb-admin
> bower install

sh: bower: command not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! dynamodb-admin@1.13.0 postinstall: `bower install`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the dynamodb-admin@1.13.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/package.json
+++ b/package.json
@@ -31,10 +31,10 @@
     "es7-object-polyfill": "0.0.1",
     "express": "^4.14.0",
     "js-yaml": "^3.6.1",
+    "bower": "^1.8.0",
     "lodash": "^4.16.4"
   },
   "devDependencies": {
-    "bower": "^1.8.0",
     "jest-cli": "^15.0.2"
   }
 }

